coeus-impl/src/main/resources/org/kuali/coeus/propdev/impl/budget/subaward/ProposalBudgetSubawardsPage.xml

```diff
@@ -192,7 +195,7 @@
 					</property>
 				</bean>
 				<bean p:actionLabel="Sync from PDF" parent="Uif-SecondaryActionButton-Mini" p:order="30" 
-					p:disabled="@{dialogDataObject == null}" p:methodToCall="syncFromPdf">
+					p:disabled="@{dialogDataObject == null}" p:render="@{!#parent.readOnly}" p:methodToCall="syncFromPdf">
```

Comment:
Ditto to above question. Why hide the button in readonly mode here, but show a disabled save button above?